plugins/modules/web_infrastructure/jira.py

```diff
@@ -694,7 +694,14 @@ def request(
             try:
                 error = json.loads(info['body'])
             except Exception:
-                self.module.fail_json(msg=to_native(info['body']), exception=traceback.format_exc())
+                msg = 'ERROR request {method} "{url}": {status} {msg}\n{body}'.format(
```

Comment:
How about:
```suggestion
                msg = 'The request "{method} {url}" returned the unexpected status code {status} {msg}\n{body}'.format(
```
It's already clear there is an error since the module failed :)

Reply:
Yup, works for me